Lesson_2/task_5.js

```diff
@@ -0,0 +1,56 @@
+// Выполнить преобразования с массивом сотрудников компании:
+// 1. Узнать среднюю зарплату сотрудников
+// 2. Отсортировать сотрудников по зарплате
+// 3. Получить список сотрудников с зарплатой >4500 и возрастом > 25 лет
+
+const employees = [
+    {
+        firstName: 'Alex',
+        lastName: 'Smith',
+        age: 54,
+        salary: 10000,
+        position: 'Architect'
+    },
+    {
+        firstName: 'Gustav',
+        lastName: 'Andersen',
+        age: 31,
+        salary: 5000,
+        position: 'Software engineer'
+    },
+    {
+        firstName: 'Liz',
+        lastName: 'Taylor',
+        age: 20,
+        salary: 7000,
+        position: 'Manager'
+    }
+]
+
+
+let averageSalary = function (employees) {
+    return employees
+        .map((employee) => employee.salary)
+        .reduce((total, emploee_salary) => total + emploee_salary) / employees.length
+}
```

Comment:
Можно исключить дополнительный перебор массива через `map()`:
```suggestion
let averageSalary = function (employees) {
    return employees
        .reduce((total, employee) => total + employee.salary, 0) / employees.length
}
```